templates/helm/templates/deployment.yaml.tpl

```diff
@@ -51,6 +51,13 @@ spec:
         - "$(AWS_REGION)"
         - --aws-endpoint-url
         - "$(AWS_ENDPOINT_URL)"
+{{- if .Values.aws.identity_endpoint_url }}
+        - --aws-identity-endpoint-url
+        - "$(AWS_IDENTITY_ENDPOINT_URL)"
+{{- end }}
+{{- if .Values.aws.allow_unsafe_aws_endpoint_urls }}
+        - --allow-unsafe-aws-endpoint-urls
+{{- end }}
```

Comment:
Thank you for your PR @aleslash. 

These need to be a template of a template like:
```
{{ "{{- if .Values.log.enable_development_logging }}" }}
        - --enable-development-logging
{{ "{{- end }}" }}
```

can you please address this?